control/src/attribute_names.h

```diff
@@ -0,0 +1,69 @@
+/* Copyright 2017 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef MIXERCONTROL_ATTRIBUTE_NAMES_H
+#define MIXERCONTROL_ATTRIBUTE_NAMES_H
+
+#include <string>
+
+namespace istio {
+namespace mixer_control {
+
+// Define attribute names
+struct AttributeName {
```

Comment:
static std::string might cause problems, we might need something like envoys `Headers::get().`, use a static pointer for that.

Reply:
I think it is safe to use static for some simple class.  such as Status here at
https://github.com/google/protobuf/blob/master/src/google/protobuf/stubs/status.h#L80

std::string is so simple,  it should be OK.   If we want to change that.  There are too many places that we need to change.

Reply:
OK, replaced std::string with char[] for attribute names.